### PR TITLE
Update Big Peer documentation

### PR DESCRIPTION
**Pull request by ergl**
Wording changes, as well as some typos

### CODE REVIEW

File: docs/common/how-it-works/big-peer.md [hunk 1]
```diff
@@ -345,7 +345,7 @@ the query. At present, Big Peer shards data by Application AND Collection
 (however this can change in the future). The coordinator will assemble a
 list of partitions needed to answer the query, and pick one replica
 from each partition. It picks the replica based on a
-[fault-detector](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.581.8294&rep=rep1&type=pdf),
+[fault-detector](https://www.researchgate.net/profile/Benjamin-Satzger/publication/4339407_A_Lazy_Monitoring_Approach_for_Heartbeat-Style_Failure_Detectors/links/0912f50bcb4b32fc08000000/A-Lazy-Monitoring-Approach-for-Heartbeat-Style-Failure-Detectors.pdf),
```
Comment (ergl) [inline docs/common/how-it-works/big-peer.md:348]: @russelldb Original link is dead, but I'm assuming you were talking about this paper. Correct me if I'm wrong, though.

Comment (ergl) [reply docs/common/how-it-works/big-peer.md:348]: Authoritative link is https://doi.org/10.1109/ARES.2008.51, but that's paywalled

**Review by russelldb — APPROVED** [review]
Many thanks for improving the doc!

**Review by okdistribute — APPROVED** [review]

**Comment by okdistribute** [comment]
Thanks!

**Pull request merged**